moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -249,18 +259,25 @@ const ComponentForm = ({
                 id="completion-date"
                 name="completionDate"
                 control={control}
-                render={({ onChange, value, ref }) => (
-                  <DatePicker
-                    inputRef={ref}
-                    value={value}
-                    onChange={onChange}
-                    clearable={true}
-                    emptyLabel="mm/dd/yyyy"
-                    format="MM/dd/yyyy"
-                    variant="outlined"
-                    label={"Completion date"}
-                  />
-                )}
+                render={({ onChange, value, ref }) => {
+                  return (
+                    <DatePicker
+                      inputRef={ref}
+                      value={parseDate(value)}
+                      onChange={(date) => {
+                        const newDate = date
+                          ? format(date, "yyyy-MM-dd OOOO")
+                          : null;
+                        onChange(newDate);
+                      }}
+                      clearable={true}
+                      format="MM/dd/yyyy"
+                      variant="outlined"
+                      label={"Completion date"}
+                      onAccept={(props) => console.log(props)}
```

Comment:
this is how I found out what the form was accepting it as, I will delete before merging.